modnet/models.py

```diff
@@ -201,7 +220,14 @@ def fit(
         x = training_data.get_featurized_df()[
             self.optimal_descriptors[:self.n_feat]
         ].values
-        y = [training_data.df_targets[targ].values.astype(np.float, copy=False) for targ in self.targets_flatten]
+
```

Comment:
Could we also default to a better loss when doing a classification here (KL or otherwise)?